lib/options.dart

```diff
@@ -57,6 +57,9 @@ class DartdocGeneratorOptionContext extends DartdocOptionContext {
   // TODO(jdkoren): duplicated temporarily so that GeneratorContext is enough for configuration.
   @override
   bool get useBaseHref => optionSet['useBaseHref'].valueAt(context);
+
+  /// The 'resourcesDir' Dartdoc option if one was specified; otherwise `null`.
```

Comment:
nit: don't we usually write dartdoc without a leading cap?

Reply:
Done